CRM/Report/Form.php

```diff
@@ -4892,4 +4892,23 @@ public function joinGroupTempTable($baseTable, $field, $tableAlias) {
     }
   }
 
+  /**
+   * Get all labels for credit card type rows in report.
+   *
+   * @param string $row
+   *   comma separated option values of option group "credit_card_type".
+   */
+  public function getGroupCreditCardType($row) {
+    $creditCardTypes = CRM_Core_PseudoConstant::get('CRM_Financial_DAO_FinancialTrxn', 'credit_card_type');
```

Comment:
The variable name should not be called $row - that isn't what you are passing $creditCardTypes or similar

@colemanw there is no helper you can think of that would make this fn unrequired is there? I feel like there should be but perhaps not